doc/man/8/mount.ceph.rst

```diff
@@ -71,6 +71,20 @@ Basic
 :command:`mount_timeout`
     int (seconds), Default: 60
 
+:command:`ms_mode=<legacy|crc|secure|prefer-crc|prefer-secure>`
```

Comment:
@idryomov : Side note, but as good a place to discuss it as any.

It occurs to me that we probably should have made this option `ms-mode` (with a hyphen instead of underscore). It's a bit odd to have the option name with an underscore and the values use hyphens. Should we change this before v5.11 ships?

Reply:
Hrm...  I tend to think of an underscore as a kind of space, whereas a hyphen is just a hyphen.  If I were describing these modes in an email, I'd probably write "prefer-crc" and "prefer-secure", so that's why it ended up this way.

As for changing, definitely not the option name -- all mount options have underscores in them.  We could change `prefer-crc` to `prefer_crc` and `prefer-secure` to `prefer_secure`, but I'm not sure it's worth it at this point.